scripts/drtprod

```diff
@@ -456,6 +456,143 @@ WantedBy=multi-user.target
+        ldr1="$(roachprod ssh drt-ldr1:1 -- ./cockroach encode-uri --inline --ca-cert ./certs/ca.crt --key ./certs/client.root.key --cert ./certs/client.root.crt {pgurl:1:system} | sed 's,postgresql://postgres://,postgres://,')"
+        $0 sql drt-ldr1:1 -- -e "CREATE EXTERNAL CONNECTION 'drt-ldr2' AS '${ldr2}'"
+        $0 sql drt-ldr2:1 -- -e "CREATE EXTERNAL CONNECTION 'drt-ldr1' AS '${ldr1}'"
+        roachprod sql drt-ldr1:1 -- -e "CREATE LOGICAL REPLICATION STREAM FROM TABLE kv.kv ON 'external://drt-ldr2' INTO TABLE kv.public.kv;"
+        roachprod sql drt-ldr2:1 -- -e "CREATE LOGICAL REPLICATION STREAM FROM TABLE kv.kv ON 'external://drt-ldr1' INTO TABLE kv.public.kv;"
+
+        echo
+        echo "Completed setup of ldr1 and ldr2, with logical data replication between them on table kv.kv."
+        echo
+        echo "Run 'drtprod create workload-ldr' after the restore completes to start the workload."
+        echo
+        ;;
+      "workload-ldr")
+        set -x
+        roachprod create workload-ldr \
+          --clouds="gce" \
+          --gce-zones="us-east1-c" \
+          --nodes="1" \
+          --gce-machine-type="n2-standard-8" \
+          --os-volume-size 100 \
+          --username workload \
+          --lifetime 8760h \
+          --gce-image "ubuntu-2204-jammy-v20240319"
+        # setup dns
+        $0 dns workload-ldr create || $0 dns workload-ldr
+
+        # push cockroach to the worker to run its built-in workloads
+        roachprod stage workload-ldr:1 cockroach
+        roachprod stage workload-ldr:1 workload
+
+        # Push certs for both clusters to the worker.
+        roachprod get drt-ldr1:1 certs certs.ldr1
+        roachprod put workload-ldr:1 certs.ldr1
+        roachprod get drt-ldr2:1 certs certs.ldr2
+        roachprod put workload-ldr:1 certs.ldr2
+        roachprod ssh workload-ldr:1 -- "chmod 0600 certs.ldr1/* certs.ldr2/*"
+        rm -rf certs.ldr1 certs.ldr2
+
+        # Setup pgurls for worklooads to use.
+        roachprod ssh workload-ldr:1 -- "
+        echo \"$(roachprod pgurl drt-ldr1 --secure | sed 's/=certs/=certs.ldr1/g' | sed "s/'//g" )\" > pgurls.ldr1.txt;
+        echo \"$(roachprod pgurl drt-ldr2 --secure | sed 's/=certs/=certs.ldr2/g' | sed "s/'//g" )\" > pgurls.ldr2.txt;
+        ln -sf pgurls.ldr1.txt pgurls.txt"
+
+        # Setup the tpcc workload runner as a script invoked by systemd.
+        roachprod ssh workload-ldr:1 -- 'cat - > kv50_run_1.sh << EOF
+#!/usr/bin/env bash
+
+exec ./cockroach workload run kv \\
+  --concurrency=32
+  --read-percent=50 \\
+  \$(cat /home/ubuntu/pgurls.ldr1.txt)
+EOF'
+        roachprod ssh workload-ldr:1 -- 'cat - > kv50_run_2.sh << EOF
+#!/usr/bin/env bash
+
+exec ./cockroach workload run kv \\
+  --concurrency=32
+  --read-percent=50 \\
+  \$(cat /home/ubuntu/pgurls.ldr2.txt)
+EOF'
+        roachprod ssh workload-ldr:1 -- "chmod +x ./kv50_run_1.sh;
```

Comment:
Nit: can we make this `kv50_run_ldr1.sh` so it's clearer what the `1` means?

Reply:
Done.